lib/src/components/NxTextInput/__tests__/NxTextInput.test.tsx

```diff
@@ -4,258 +4,518 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React, { createRef } from 'react';
-import { faCheck, faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+import React, { RefAttributes } from 'react';
+import { render, within } from '@testing-library/react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import userEvent from '@testing-library/user-event';
 
-import NxTextInput, { PrivateNxTextInput, Props } from '../NxTextInput';
-import { mount } from 'enzyme';
-import { FormAriaContext } from '../../NxForm/context';
+import NxTextInput, { PublicProps } from '../NxTextInput';
+import NxForm from '../../NxForm/NxForm';
 
 describe('PrivateNxTextInput', function() {
```

Comment:
`PrivateNxTextInput` isn't what's being tested here, and doesn't need to be directly tested since it's not exposed outside of the RSC library.
```suggestion
describe('NxTextInput', function() {
```